includes/class-evf-form-task.php

```diff
@@ -296,6 +296,12 @@ public function do_task( $entry ) {
 			$response_data['message']  = $message;
 			$response_data['response'] = 'success';
 
+			if ( isset( $this->form_data['settings']['redirect_to'] ) && '2' === $this->form_data['settings']['redirect_to'] ) {
```

Comment:
@coderkoala Let's get rid of integer to define the actual setting for `redirect_to`!

Reply:
Resolved with f0c2245.